.github/workflows/ci.yml

```diff
@@ -137,6 +137,19 @@ jobs:
             source presto-product-tests/conf/product-tests-${{ matrix.config }}.sh &&
               presto-hive-hadoop2/bin/run_hive_abfs_access_key_tests.sh
           fi
+      - name: Run Hive Azure ABFS OAuth Tests
+        if: matrix.config != 'config-empty' # Hive 1.x does not support Azure storage
+        env:
+          ABFS_CONTAINER: ${{ secrets.AZURE_ABFS_CONTAINER }}
+          ABFS_ACCOUNT: ${{ secrets.AZURE_ABFS_ACCOUNT }}
+          ABFS_OAUTH_CLIENT_ENDPOINT: ${{ secrets.AZURE_ABFS_OAUTH_CLIENT_ENDPOINT }}
```

Comment:
We can shorten these names to drop the redundant "client" part and make them more consistent with the others
```
AZURE_ABFS_OAUTH_ENDPOINT
AZURE_ABFS_OAUTH_CLIENTID
AZURE_ABFS_OAUTH_SECRET